### PR TITLE
new file:   packages/parany/parany.14.0.1/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
bug correction: the bug could be triggered on many core computers cf. https://github.com/UnixJunkie/pardi/issues/25

### CODE REVIEW

File: packages/parany/parany.14.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+depends: [
+  "base-unix"
+  "cpu"
+  "dune" {>= "1.6.0"}
+  "ocaml" {>= "4.03.0"}
+]
+available: os != "win32"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-p" name "-j" jobs "src/test.exe"] {with-test}
```
Comment (mseri) [inline packages/parany/parany.14.0.1/opam:17]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs "src/test.exe"] {with-test & os != "macos"}
```
Due to
```
#=== ERROR while compiling parany.14.0.1 ======================================#
# context              2.2.0~alpha2 | macos/x86_64 | ocaml-base-compiler.4.14.1 | pinned(https://github.com/UnixJunkie/parany/archive/v14.0.1.tar.gz)
# path                 ~/.opam/4.14.1/.opam-switch/build/parany.14.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build ./test.sh
# exit-code            2
# env-file             ~/.opam/log/parany-30775-6a6430.env
# output-file          ~/.opam/log/parany-30775-6a6430.out
### output ###
# ./test.sh: line 7: _build/default/src/test_parmap.exe: No such file or directory
# diff: /dev/fd/63: Operation not permitted
# diff: /dev/fd/63: Operation not permitted
# diff: /dev/fd/63: Operation not permitted
# diff: /dev/fd/63: Operation not permitted
```

**Comment by UnixJunkie** [comment]
I don't see why the test exe would not build on OS X though (I just tried and the 'make test' target just works in there).

**Comment by mseri** [comment]
I think it is an issue with the sandbox, does it work with `opam install -t parany`?

File: packages/parany/parany.14.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+depends: [
+  "base-unix"
+  "cpu"
+  "dune" {>= "1.6.0"}
+  "ocaml" {>= "4.03.0"}
+]
+available: os != "win32"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-p" name "-j" jobs "src/test.exe"] {with-test & os != "macos"}
```
Comment (mseri) [inline packages/parany/parany.14.0.1/opam:17]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs "src/test.exe"] {with-test}
```

**Comment by mseri** [comment]
I did disable the wrong test though

**Comment by UnixJunkie** [comment]
'opam install -t parany' does work on a Mac.
Even if I opam pin the package to the git repository or parany.

**Comment by mseri** [comment]
Ok, then I will revert the changes. It is either a difference within opam 2.2 or a CI setup issue.

**Comment by mseri** [comment]
Thanks for checking

File: packages/parany/parany.14.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "base-unix"
+  "cpu"
+  "dune" {>= "1.6.0"}
+  "ocaml" {>= "4.03.0"}
+]
+available: os != "win32"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-p" name "-j" jobs "src/test.exe"] {with-test}
+  ["./test.sh"] {with-test & os != "macos"}
```
Comment (mseri) [inline packages/parany/parany.14.0.1/opam:18]:
```suggestion
  ["./test.sh"] {with-test}
```

**Comment by mseri** [comment]
Thanks

**Comment by UnixJunkie** [comment]
Thanks Marcello!

**Pull request merged**